src/app/item-page/orcid-page/orcid-auth/orcid-auth.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="container custom-accordion mb-4">
+    <ngb-accordion activeIds="auth">
+        <ngb-panel title="{{'person.orcid.registry.auth' | translate}}" id="auth">
+        <ng-template ngbPanelContent>
+            <div class="container">
+            <ng-container *ngIf="isLinkedToOrcid(); then orcidLinked; else orcidNotLinked"></ng-container>
+            </div>
+        </ng-template>
+        </ngb-panel>
+    </ngb-accordion>
+  
+    <ng-template #orcidLinked>
+        <div class="row mt-3">
+        <div *ngIf="getOrcidAuthorizations().length > 0" class="card m-2 col-md p-0">
+            <div class="card-header">{{ 'person.page.orcid.granted-authorizations'| translate }}</div>
+            <div class="card-body">
+            <div class="container">
+                <ul>
+                <li *ngFor="let auth of getOrcidAuthorizations()"> {{getAuthorizationDescription(auth) | translate}} </li>
+                </ul>
+            </div>
+            </div>
+        </div>
+        <div class="card m-2 col-md p-0">
+            <div class="card-header">{{ 'person.page.orcid.missing-authorizations'| translate }}</div>
+            <div class="card-body">
+            <div class="container">
+                <div *ngIf="( missingAuthorizations$ | async ).length === 0" class="p-3 mb-2 alert alert-success rounded">
+                {{'person.page.orcid.no-missing-authorizations-message' | translate}}
+                </div>
+                <div *ngIf="( missingAuthorizations$ | async ).length > 0" class="p-3 mb-2 alert alert-warning rounded" >
+                {{'person.page.orcid.missing-authorizations-message' | translate}}
+                <ul>
+                    <li *ngFor="let auth of ( ( missingAuthorizations$ | async ) )"> {{getAuthorizationDescription(auth) | translate }} </li>
+                </ul>
+                </div>
+            </div>
+            </div>
+        </div>
+        </div>
+        <div *ngIf="(onlyAdminCanDisconnectProfileFromOrcid() | async)" class="row mt-3 p-3 mb-2 bg-info text-white rounded d-flex justify-content-center">
```

Comment:
It is bad for performance to call methods from a template. They will be called multiple times a second, any time Angular checks anything about the template basically. 

If the methods are trivial such as `getOrcidAuthorizations()` which only checks a metadata value it's not that big a deal (though you could still improve performance by storing their result in a component property, especially if you use them more than once). 

However `onlyAdminCanDisconnectProfileFromOrcid()` and `ownerCanDisconnectProfileFromOrcid()` ultimately depend on a backend call, so these should really be removed from the template